Packs/CortexXpanse/ReleaseNotes/1_2_10.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Xpanse Feed
+
+- Added support for **Indicators types to fetch** parameter that allows selection of indicator types to fetch in the feed.
```

Comment:
Added the *Indicator types to fetch* parameter to enable selecting indicator types to fetch in the feed.